csharp/src/elements/Geometry/Polygon.cs

```diff
@@ -94,29 +94,6 @@ public bool Contains(Vector3 point)
             return true;
         }
 
-        /// <summary>
-        /// Tests if all the supplied Vector3 points are within this Polygon without coincidence with an edge when compared on a shared plane.
-        /// </summary>
-        /// <param name="points">The list of Vector3 points to compare to this Polygon.</param>
-        /// <returns>
-        /// Returns true if all the supplied Vector3 points are within this Polygon when compared on a shared plane. Returns false if any of the Vector3 points are outside this Polygon or if the supplied Vector3 point list is null.
-        /// </returns>
-        public bool Contains(IList<Vector3> points)
```

Comment:
Thank you! There's a better than even chance that we'll put these back in some form one day. But I want to be sure when we do that it's a consistent design.

Reply:
Yeah, I realized you were right, this should be a library-wide approach if / when we do it. I only need one of the plural functions for now, so I'll instantiate it in my HyparSpaces library as a Polygon extension. However, I need all of the single inputs in various bit of HyparSpaces.